tests/unittest_checker_similar.py

```diff
@@ -5,18 +5,20 @@
 # Copyright (c) 2016-2017 Claudiu Popa <pcmanticore@gmail.com>
```

Comment:
Clearly this file could do with some re-factoring, but the diff is minimised with this specific patch for the intended outcome. Once this PR is accepted I am very happy to re-factor and modernise the file.